src/coreclr/src/pal/src/libunwind_mac/src/missing-functions.c

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
I wonder if this should use libunwind's license header.

Reply:
I'm not sure either.  I did copy some of those function from the libunwind source.

Reply:
I removed the MIT header and added the libunwind one.